File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        vector<int> kLarg;
        for (int i = 0; i < k; ++i) {
            kLarg.push_back(nums[i]);
        }
        sort(kLarg.begin(), kLarg.end());
        for (int i = k; i < nums.size(); ++i) {
            if (nums[i] > kLarg[0]) {
                kLarg[0] = nums[i];
                sort(kLarg.begin(), kLarg.end());
            }
        }
        return kLarg[0];
    }
};

File: 215-kth-largest-element-in-an-array/215-kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        largest = [float('-inf')] * k
        for num in nums:
            if num > largest[0]:
                largest[0] = num
                largest.sort()
        return largest[0]
        